### PR TITLE
fix typos and misspellings

### PR DESCRIPTION
**Pull request by greggdourgarian**
fix typos and misspellings

### CODE REVIEW

**Comment by robotally** [comment]
| Vote | Count | Reviewers |
| :---: | :---: | :---: |
| :+1: | 0 |  |
| :-1: | 0 |  |

_Updated: Sun Aug 28 03:29:44 UTC 2016_

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 6b8b198284652b0bcdc5d51816c9390cd1af5bd0
  * Commits must be prefixed with the package(s) they modify

 * Commit: 7cd7990904362b7ba6db6c7dddaa6e9fb4e6dcac
  * Commit subjects should be kept under 80 characters

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

File: Godeps/_workspace/src/gopkg.in/fatih/set.v0/set_ts.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ type Set struct {
 	l sync.RWMutex // we name it because we don't want to expose it
 }
 
-// New creates and initialize a new Set. It's accept a variable number of
+// New creates and initialize a new Set. It accepts a variable number of
```
Comment (karalabe) [inline Godeps/_workspace/src/gopkg.in/fatih/set.v0/set_ts.go]: Please revert this change. This is a vendored library from an upstream repo. We don't make modifications to it.

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 6b8b198284652b0bcdc5d51816c9390cd1af5bd0
  * Commits must be prefixed with the package(s) they modify

 * Commit: 7cd7990904362b7ba6db6c7dddaa6e9fb4e6dcac
  * Commit subjects should be kept under 80 characters

 * Commit: 3b550c5366e5cc116fcffaa1f18e365d3f8a06ed
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

File: Godeps/_workspace/src/gopkg.in/fatih/set.v0/set_ts.go (hunk 1 above)
Comment (greggdourgarian) [inline Godeps/_workspace/src/gopkg.in/fatih/set.v0/set_ts.go]: ok, reverted via git checkout 702218008ee2b6d708d6b2821cdef80736bb3224  Godeps/_workspace/src/gopkg.in/fatih/set.v0/set_ts.go on latest commit..tx!

**Comment by zelig** [comment]
please squash to one commit otherwise :+1:

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 6b8b198284652b0bcdc5d51816c9390cd1af5bd0
  * Commits must be prefixed with the package(s) they modify

 * Commit: 7cd7990904362b7ba6db6c7dddaa6e9fb4e6dcac
  * Commit subjects should be kept under 80 characters

 * Commit: 3b550c5366e5cc116fcffaa1f18e365d3f8a06ed
  * Commits must be prefixed with the package(s) they modify

 * Commit: e0ea6ac3e0a798bc259dee5cee0b0281e5866f94
  * Commit subjects should be kept under 80 characters

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

**Comment by greggdourgarian** [comment]
ok i've squashed the commit in question...tx!

**Comment by fjl** [comment]
This has gotten out of date again because we didn't merge it in time. I will apply the fixes for correct use of "its" vs "it's" in a follow-up PR. Thanks!
